### PR TITLE
Shifting testing

### PR DESCRIPTION
**Pull request by erinyoung**
# BWA HAHAHA!

😈😈😈😈😈😈😈😈😈😈 

I HAVE MADE THE `test-tool.yml` FILES COMPLETELY UNNCESSARY!

VICTORY!!!

😈😈😈😈😈😈😈😈😈😈

In this PR, here's what I've done:
1. I deleted all the unnecessary `test-tool.yml` and `deploy-tool.yml` files and the test template
2. I added in two new github actions workflows to  .github/workflows/ 
    1. a manual way to test tools : `Manual-test.yml`
    2. a workflow to automatically find and test new dockefiles with each PR : `test-PR-dockerfiles.yml`
3. I changed the name of Manual_deploy.yml to Manual-deploy.yml so that it'd match the other workflows
4. I changed all the documentation that I could find to remove any language about create test-tool.yml files. I also updated some of the language in those files. I'm fairly certain that I've missed some, though, so let me know if there's more I need to add.
5. I got rid of contact info in the issue template. It was in the mulitqc repo, so I thought it would be more useful that it was
6. I simplified the PR template so that it's much shorter and simpler

### CODE REVIEW

**Comment by lskatz** [comment]
Smart.  I just want to comment to follow this.  I thought using the `strategy` was a good technique in combination with finding new files.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -10,65 +10,20 @@ Documentation on how to contribute can be found at https://staphb.org/docker-bui
...
 -->
 
+Pull Request (PR) checklist:
+- [ ] Include a description of what is in this pull request in this message.
+- [ ] The dockerfile successfully builds to a test target for the user creating the PR. (i.e. `docker build --tag samtools:1.15test --target test docker-builds/samtools/1.15` )
+- [ ] Directory structure as name of the tool in lower case with special characters removed with a subdirectory of the version number (i.e. `spades/3.12.0/Dockerfile`)
+   - [ ] (optional) All test files are located in same directory as the Dockerfile (i.e. `shigatyper/2.0.1/test.sh`)
+- [ ] Create a simple container-specific [README.md](https://github.com/StaPH-B/docker-builds/blob/master/.github/workflow-templates/readme-template.md) in the same directory as the Dockerfile (i.e. `spades/3.12.0/README.md`)
+   - [ ] If this README is longer than 30 lines, there is an explanation as to why more detail was needed
+- [ ] Dockerfile includes the recommended [LABELS](https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile)
```
Comment (kapsakcj) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: please change `https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile` to `https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18` for fancy permalinking!

Comment (kapsakcj) [inline .github/PULL_REQUEST_TEMPLATE.md:24]: please change `https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile` to `https://github.com/StaPH-B/docker-builds/blob/master/dockerfile-template/Dockerfile#L8-L18` for fancy permalinking!

**Comment by kapsakcj** [comment]
`.github/workflows/test-PR-dockerfiles.yml` has worked great in my testing on my fork. No complaints here. Only thing to remember is that users will have to write dockerfiles to our standards with the `app` and `test` layers in order for it to work.

Will soon test out `.github/workflows/manual-test.yml` and provide feedback.

Then I'll review the documentation changes and do a final lookover before approving.

Really awesome work, Erin. Can't wait to get this merged

**Review by kapsakcj — APPROVED** [review]
All looks good to me. We may encounter some hiccups after merging, but everything worked well in my testing! I'm going to go ahead and merge!

**Pull request merged**